specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/OracleAzureDbPostgreSqlSyncTask.json

```diff
@@ -97,6 +97,120 @@
           "description": "Target server brand version",
           "readOnly": true
         },
+        "validationErrors": {
+          "description": "Validation errors associated with the task",
+          "type": "array",
+          "items": {
+            "$ref": "./Common.json#/definitions/ReportableException"
+          },
+          "readOnly": true
+        },
+        "databaseSchemaMap": {
+          "type": "array",
+          "description": "Mapping of schemas per database",
+          "items": {
+            "type": "object",
+            "properties": {
+              "database": {
+                "type": "string"
+              },
+              "schemas": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "GetUserTablesOracleTaskInput": {
+      "type": "object",
+      "description": "Input for the task that gets the list of tables contained within a provided list of Oracle schemas.",
+      "properties": {
+        "connectionInfo": {
+          "$ref": "./Common.json#/definitions/OracleConnectionInfo",
+          "description": "Information for connecting to Oracle source"
+        },
+        "selectedSchemas": {
+          "type": "array",
+          "description": "List of Oracle Schemas for which to collect tables",
+          "items": {
+            "type": "string"
+          }
+        }
+      },
+      "required": [
+        "connectionInfo",
+        "selectedSchemas"
+      ]
+    },
+    "GetUserTablesOracleTaskOutput": {
```

Comment:
Missing change - Where are these being referred?

Reply:
Are you referring to the missing parts in Tasks.json? If so, these have been added now.